hedera-node/src/main/java/com/hedera/services/txns/crypto/validators/AllowanceChecks.java

```diff
@@ -198,4 +204,11 @@ default boolean validOwner(final MerkleUniqueToken nft,
 				? ownerAccount.equals(token.getTreasury())
 				: listedOwner.equals(ownerAccount.getId().asEntityId());
 	}
+
+	/**
+	 * Check if the allowance feature is enabled
+	 *
+	 * @return true if the feature is enabled in {@link com.hedera.services.context.properties.GlobalDynamicProperties}
+	 */
+	boolean isEnabled();
```

Comment:
can be renamed to `isAllowanceEnabled()`

Reply:
Done

Reply:
Since this is specific to `Allowances` I thought `isEnabled` will not need Allowance again